src/app/downloads/downloads.component.css

```diff
@@ -13,6 +13,7 @@ prx-spinner >>> .spinner::before, prx-spinner >>> .spinner::after {
   padding: 10px;
 }
 .content {
+  position: relative;
```

Comment:
#35 centers loading spinner, but does not make it more visible, so this ticket it still in the Backlog